README.md

```diff
@@ -1,12 +1,39 @@
-<h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
-<p align="center"><i>The source code for Kubesimplify.com</i></p>
+<p style="text-align:center;" align="center"><a href="https://kubesimplify.github.io/website/"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/kubesimplify/branding/main/assets/svg/horizontal/white.svg"  width="70%" align="center" style="margin-bottom:20px;">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/kubesimplify/branding/main/assets/svg/horizontal/black.svg" width="70%" align="center" style="margin-bottom:20px;">
+  <img alt="Shows an illustrated light mode kubesimplify logo in light color mode and a dark mode kubesimplify logo dark color mode." src="https://raw.githubusercontent.com/kubesimplify/branding/main/assets/svg/horizontal/white.svg" width="70%" align="center" style="margin-bottom:20px;">
+</picture></a><br /><br /></p>
+<p align="center">
+
+
+<a href="https://t.co/9KuC1EMxWo" alt="Discord Invite">
+  <img src="https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white" /></a>
+<a href="https://www.linkedin.com/company/kubesimplify/" alt="Linkedin">
+  <img src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" /></a>
+  <a href="https://twitter.com/intent/follow?screen_name=kubesimplify" alt="Twitter Follow">
+  <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" /></a> 
+  <a href="https://kubesimplify.com/" alt="Youtube">
+  <img src="https://img.shields.io/badge/Kubesimplify-%3CCOLOR%3E?style=for-the-badge&logo" /></a>
+    <a href="https://www.youtube.com/c/saiyam911" alt="Youtube">
+  <img src="https://img.shields.io/badge/YouTube-FF0000?style=for-the-badge&logo=youtube&logoColor=white" /></a>
+  </p>
+
+<p align="center"><i>If you like Kubesimplify, please ★ this repository to show your support! 🤩</i></p>
 
 This repository contains the assets required to build the Kubesimplify website and documentation. We're glad that you want to contribute! 
 
-- [Code of Conduct](code-of-conduct.md)
+## <h1>Code of conduct</h1>
```

Comment:
Remove this part please

Reply:
But then it will look like Kubesimplify is a part of cncf but we in reality Kubesimpilify just follows cncf code of conduct

Reply:
we need a code of conduct however that must be our own and not CNCFs so remove the code of conduct part for now and in future will make one(If you want to work you can take that task, sit one of the core members of kubesimplify and decide a code of conduct )

Reply:
I would love to work on that but for now I will remove code of conduct.